consumer/src/main/kotlin/streams/kafka/KafkaEventSink.kt

```diff
@@ -82,7 +87,7 @@ class KafkaEventSink: StreamsEventSink {
                                 .mapKeys { it.key.toString() }
                     }
             if (list.isNotEmpty()) {
-                log.debug("Sending data to topic $it, with data $list")
+                log.debug("Reading data from topic $it, with data $list")
```

Comment:
can we wrap all the log.debug calls in the codebase with `log.isDebugEnabled()` ?